x-pack/plugins/security_solution/public/common/components/alert_count_by_status/components/multiselect_popover/multiselect_popover.tsx

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useMemo, useState } from 'react';
+
+import { EuiFilterButton, EuiFilterSelectItem, EuiPopover } from '@elastic/eui';
+import { euiStyled } from '@kbn/kibana-react-plugin/common';
+
+export interface MultiSelectPopoverProps {
+  title: string;
+  allItems: string[];
+  selectedItems: string[];
+  onSelectedItemsChange: (newItems: string[]) => void;
+}
+
+const StyledFilterButton = euiStyled(EuiFilterButton)`
+  border-radius: 5px;
+  border: 1px solid rgba(17,43,134,0.1); 
+`;
```

Comment:
Why do we need these styles? Doesn't the `EuiFilterButton` already bring rounded corners and borders by default? https://elastic.github.io/eui/#/forms/filter-group#filter-buttons

Reply:
You would think so :P.  Though maybe there is something different about my implementation that didnt show them. below I have a with and without additional styles. 

 


| without      | with |
| ----------- | ----------- |
| <img width="163" alt="image" src="https://user-images.githubusercontent.com/28942857/191271323-84065e6d-99e8-4965-a756-3976e82ddb51.png">     | <img width="138" alt="image" src="https://user-images.githubusercontent.com/28942857/191271603-ede7365a-e0b4-4c66-b5b5-2b365fb86d6f.png">     |

What I found is that without my additional styles to override there is a class that removes these stylings:

<img width="599" alt="image" src="https://user-images.githubusercontent.com/28942857/191272736-402bfa84-e9a1-44c4-bee2-d9fe28a081a1.png">

`euiFilterButton` class is applied directly to the button and I didnt find props to override this behaviour 

Maybe this is a bug?

Reply:
🤦‍♂️

Reply:
Thanks! Now it is fixed :D